### ISSUE TITLE
Have anyone had any success with the install of Concerto2 on Ubuntu 18.04 or Debian 8 or 9

### ISSUE THREAD

**Issue by bossbarnes07** [concerto/concerto#1496]
Have attempted to install Concerto2 on both Ubuntu 18.04 and 18.10 both LTS and Debian both have problems with the install missing packages. Can anyone provide me a work around  beside the vm option. Thank you.

**Comment by Kohlmeyer** [comment]
I also am having a really difficult time getting a running version of the ova or cloning Concerto 2 correctly. I would really like to see if there are solutions to this issue as well, thanks!

**Comment by Ahmedjaseemiq** [comment]
me too I cant find any installation step for any Ubuntu or Debian version can work today, actually I start doubt  if there Concerto work!!, even Youtube I cant find.

**Comment by mzocher** [comment]
I was able to get my server upgraded from Ubuntu LTS 16.04 to 18.04, but because it ships with Ruby 2.5 it took some tweaks:

- Before install, needed to explicitly install the older version of bundle.  Ran the command `gem install bundler -v 1.16.1`
- Needed to use the delayed_job gem version 4.14, as it adds Ruby 2.5 support (by switching from yaml_as to yaml_tag - see [https://github.com/collectiveidea/delayed_job/blob/master/CHANGELOG.md](url). Edited Gemfile.lock and changed the delayed_job version from 4.1.3 to 4.1.4
- Devise 3.5.10 has a bug that was fixed, but a new version was never replaced (see [https://github.com/plataformatec/devise/issues/4883].  You can point to the stable v 3 branch and get that new patch.  To do so, open Gemfile and replace `gem 'devise', '~> 3.5.5'` with `gem 'devise', :git=> "https://github.com/plataformatec/devise.git", :branch => "3-stable"`
-    Finally. ran dpkg --configure concerto-full to make sure everything was configured properly.

**Comment by GrayMatter** [comment]
Came across this in the Concerto forum.  I haven't tried it but plan to give it a shot in the next week or two.

https://groups.google.com/d/msg/concerto-digital-signage/ywa9exSeA74/3JC6FsHdAQAJ

**Comment by paulfurze** [comment]
Almost !!! .... but nope :(
Follwed this...
https://groups.google.com/forum/#!topic/concerto-digital-signage/ywa9exSeA74
but it eventually fails at...

`sudo dpkg --configure concerto-full`

Then the below happens.
I've no experience of ruby/rails/gems. Any pointers would be well received.

```
Bundler could not find compatible versions for gem "bundler":
  In Gemfile:
    bundler-audit was resolved to 0.6.0, which depends on
      bundler (~> 1.2)

    rails (~> 4.2.6) was resolved to 4.2.9, which depends on
      bundler (< 2.0, >= 1.3.0)

  Current Bundler version:
    bundler (2.0.2)
This Gemfile requires a different version of Bundler.
Perhaps you need to update Bundler by running `gem install bundler`?

Could not find gem 'bundler (< 2.0, >= 1.3.0)', which is required by gem 'rails
(~> 4.2.6)', in any of the sources.

dpkg: error processing package concerto-full (--configure):
 subprocess installed post-installation script returned error exit status 6
```
I have removed bundler 2.02, re-installed 1.16 but all that happens is that v2 is just re-installed.

**Comment by mfrederickson** [comment]
Version 2.3.7 has the changes mentioned above (thanks @mzocher) and will be packaged soon for xenial, buster, and bionic. In the meantime, you can grab master via the [docker instructions](https://github.com/concerto/concerto#docker-image) here and have it up and running in minutes.
